packages/public/client/src/feature-prototypes/donation-profiles/index.ts

```diff
@@ -0,0 +1,205 @@
+import { getAsset } from '../../frontend/modules/assets'
+import { getAuthenticatedUserID } from '../../frontend/modules/user'
+import { activeAuthors } from './active-authors'
+import { activeDonors } from './active-donors'
+import { activeReviewers } from './active-reviewers'
+
+const donorsSpec = {
+  userList: activeDonors,
+  img: 'donors.png',
+  imgBig: 'big-donor.png',
+  otherUserProfileMessage:
+    '%username% trägt mit einer regelmäßigen Spende dazu bei, dass serlo.org komplett kostenlos, werbefrei und unabhängig ist. <a style="text-decoration: underline;" href="/user/me#spenden">Kannst auch du dir vorstellen, uns mit einem kleinen Betrag zu unterstützen?</a>',
+  ownProfileMessage:
+    'Wir sind die ersten %no% Pioniere beim Aufbau einer langfristigen und unabhängigen Finanzierung für serlo.org.'
+}
+
+const userProfileSpecs: UserProfileSpec[] = [
+  {
+    userList: activeReviewers,
+    img: 'reviewers.png',
+    imgBig: 'big-reviewer.png',
```

Comment:
In the long-term, these should be SVGs, if possible

Reply:
Yes, that's true. -> https://github.com/serlo/serlo.org/issues/375